scripts/cleanup_openstack_instance_snapshot.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+"""Cleanup unused instance snapshot from glance repository
+
+Usage: scripts/cleanup_openstack_instance_snapshot.py [optional list of provider keys]
+
+If no providers specified, it will cleanup all of them.
+
+"""
+import sys
+from datetime import datetime
+from datetime import timedelta
+
+import iso8601
+import tzlocal
```

Comment:
If I am not wrong you need L14,L15 imports to be before those that are `from...`

Reply:
No, we cant move it  as the datetime is in built pyton library, so it should come first where as iso8601 and tzlocal are 3rd-party library, so it should come after that.

Reply:
Got it.